src/test/resources/log4j2.xml

```diff
@@ -1,13 +1,63 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <Configuration status="info">
+	<Properties>
+		<Property name="baseDir">src/test/resources/logs</Property>
+		<Property name="logPattern">%d{YYYY-MM-dd HH:mm:ss.SSS} [%t] %-5level %l - %msg%n</Property>
+	</Properties>
 	<Appenders>
 		<Console name="Console" target="SYSTEM_OUT">
-			<PatternLayout pattern="%d{YYYY-MM-dd HH:mm:ss.SSS} [%t] %-5level %logger{36} - %msg%n" />
+			<PatternLayout pattern="${logPattern}" />
 		</Console>
+
+		<RollingFile
+			name="mainLogFile"
+			fileName="${baseDir}/Main-${date:MM-dd-yyyy_HH.mm.ss}.log"
+			filePattern="${baseDir}/Main-%d{MM-dd-yyyy_HH.mm.ss}.log"
+		>
```

Comment:
Just curious: is IntelliJ doing this formatting?

Reply:
No, I've been formatting this way.  It's a convention that I've seen used in many places when you have several attributes with long values, they are moved to one on each line.

Reply:
Huh. The ">" dangling on its own line is a little weird looking to me. I don't think I've seen that before.

Reply:
Yeah, I think I picked up that way of doing it by watching React tutorials that uses a kind of embedded HTML in JavaScript called JSX.  

The other way people look to be doing it is to have the ">" on the same line as the last attribute:
https://stackoverflow.com/questions/14791592/break-line-on-each-tag-attribute-and-keep-them-aligned-in-visual-studio-html-cod

I can do it that way from now on if you prefer.

Reply:
Okay, sounds good.  Here's an example on the JSX attribute formatting: https://github.com/prettier/prettier/issues/5501.